README.md

```diff
@@ -1,2 +1,75 @@
-# turtle
-The fastest agile project management app
+# Turtle
+
+Turtle is an issue tracker for software developers.
+
+- easy to use (tasks are controlled by dragging and dropping them around)
+- built for agile development methodology
+- self-sufficient (does not require third-party services to function)
+- free and open-source
+
```

Comment:
We might want some screenshots and description of how this works. But that can safely be added on Wednesday/Thursday/Friday :-)